### PR TITLE
Cockpit 163

### PR DESCRIPTION
**Pull request by martinpitt**
/!\ DRAFT, not released yet.

Rendered preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-163.html

### CODE REVIEW

**Comment by garrett** [comment]
OK, submitted a few commits as suggestions. Feel free to revert any you don't like and/or stack your own changes on top, then merge it all together into one.

**Comment by martinpitt** [comment]
Thanks @garrett, these all sound nice! Squashed.

File: _posts/2018-03-07-cockpit-163.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 163.
+
+### Hide Docker storage pool reset button when it cannot work properly
+
+Fedora Atomic used to configure a separate logical volume called
+`docker-root-lv` as a Docker storage pool. Recently, Fedora Atomic switched to
+using one large volume for both the root partition and Docker.
+
+Due to this change in Fedora Atomic, Cockpit cannot remove and free previously
```
Comment (stefwalter) [inline _posts/2018-03-07-cockpit-163.md]: How about ```s/cannot/can no longer/```

File: _posts/2018-03-07-cockpit-163.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+added storage devices. As a result, we have removed the “Reset” button on the
+storage pool page under these circumstances.
+
+### Drop "Transfer data asynchronously" VDO option on Storage page
+
+Recent versions of VDO now automatically detect the correct setting for
```
Comment (stefwalter) [inline _posts/2018-03-07-cockpit-163.md]: VDO needs to be either defined or linked, or both.

File: _posts/2018-03-07-cockpit-163.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+asynchronous data transfer, so this option is no longer needed.
+
+Manually setting the option may also lead to unsupported configurations in the
+future.
+
+*[VDO]: virtual data optimizer
```
Comment (stefwalter) [inline _posts/2018-03-07-cockpit-163.md]: Ah here it is, lets turn this into a link.

File: _posts/2018-03-07-cockpit-163.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+announced in [Cockpit 161](http://cockpit-project.org/blog/cockpit-161.html).
+All known consumers of this deprecated usage have been updated.
+
+If you have a private cockpit extension that currently includes
+`"../base1/jquery.js"`, please verify that your project still works, remove the
+reference to Cockpit’s bundled jQuery, and include your own copy of jQuery
+through NPM or a CDN.
+
+*[NPM]: Node Package Manager
+*[CDN]: Content Delivery Network
```
Comment (stefwalter) [inline _posts/2018-03-07-cockpit-163.md]: Again, lets make these links.

**Pull request merged**